.github/workflows/publish-to-pypi.yml

```diff
@@ -22,6 +22,7 @@ jobs:
           pip install pytest
           pip install zarr
           pip install setuptools wheel twine build
+          pip install -e .[test_core]
           pip install -e .
```

Comment:
Isn't this redundant after btw?

Reply:
no, otherwise it doesn't install psutils

Reply:
(that was the first failure)

Reply:
I meant the second one. Line 26.

Reply:
sorry. right!